lib/registry/config.go

```diff
@@ -51,6 +51,7 @@ type Config struct {
 	Retries       int           `yaml:"retries" json:"retries"`
 	RetryInterval time.Duration `yaml:"retry_interval" json:"retry_interval"`
 	RetryBackoff  float64       `yaml:"retry_backoff" json:"retry_backoff"`
+	BackoffMax    time.Duration `yaml:"backoff_max" json:"backoff_max"`
```

Comment:
Can we map this to retry_backoff_max and RetryBackoffMax, just so it is more consistent with above settings that are prefixed with retry_ and the option passed to httputil?

Reply:
Sure, let me rename it